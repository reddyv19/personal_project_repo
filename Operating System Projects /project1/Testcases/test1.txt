1

echo 0

0 0
