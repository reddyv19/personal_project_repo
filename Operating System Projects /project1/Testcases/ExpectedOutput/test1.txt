20650 20649 echo 0
